data-builder/parse_items.py

```diff
@@ -155,6 +169,39 @@ def get_items_from_page(itemPageURL, sets):
                 item['crafting'].append(affix['name'])
                 remove.append(affix)
 
+            # if enhancement bonus found on item we may need to translate that enhancement bonus to something else
+            if affix['name'] == 'Enhancement Bonus':
+
+                # create some booleans to reduce duplication and increase readability
+                isArmor = False
+                isShield = False
+                isWeapon = False
+
+                # assume that all item types that go in to the armor slot are armors
+                if item['slot'] == 'Armor':
+                    isArmor = True
+
+                if ((item['type'] == 'Bucklers') or \
+                    (item['type'] == 'Large shields') or \
+                    (item['type'] == 'Small shields') or \
+                    (item['type'] == 'Tower shields')):
+                    isShield = True
+
+                # for armor and shield items - enhancement bonus becomes an enhancement type bonus to armor class
+                if isArmor or isShield:
+                    affix['name'] = 'Armor Class'
+
+                # assume that every item in your weapon slot that is not a shield and is not an orb is a weapon
+                # for weapon items - enhancement bonus becomes an enhancement type bonus to accuracy and damage
+                if ((item['slot'] == 'Weapon') or (item['slot'] == 'Offhand')) \
+                    and not (isShield or item['type'] == 'Orbs'):
+                    affix['name'] = 'Accuracy'
```

Comment:
I'm wondering if it would be more clear to create `Weapon Enhancement` and `Armor Enhancement` affix groups that contain `Damage` + `Accuracy` and `Armor Class` respectively, and add those as affixes, instead of adding both `Damage` and `Accuracy`, for example, to every weapon. It seems a little confusing to me to have those two show up everywhere.

Reply:
Yeah, I've been going back and forth on reducing affixes as well. Maybe it'd be nice to either have tooltips for compound affixes, or have a checkbox to show individual affixes. The checkbox would get messy with things like Well Rounded though.

I just noticed that `Damage` should probably be a synonym of `Deadly` but we can put this in for now.